packages/features/settings/BookerLayoutSelector.tsx

```diff
@@ -0,0 +1,192 @@
+import * as RadioGroup from "@radix-ui/react-radio-group";
+import { Trans } from "next-i18next";
+import Link from "next/link";
+import { useCallback, useState } from "react";
+import { Controller, useFormContext } from "react-hook-form";
+
+import { classNames } from "@calcom/lib";
```

Comment:
Abstracted this file into a separate component so we can reuse it both in the event-types as well as in the appearance settings of the profile.